antidox/README.md

```diff
@@ -70,3 +70,23 @@ In order to run wikiwatcher on a Google Cloud Instance to retrieve all revisions
                  pip3 install mwparserfromhell
      pip3 install bs4
 ```
+# Running wikimedia
+
+1. Download and extract the [zip file](https://www.mediawiki.org/wiki/Manual:Pywikibot/Installation#Configure_Pywikibot) from the section *Install Pywikibot*
+
+2. Follow pathway to core_stable 
+
+3. Run `python3 generate_family_file.py` to create a new family
+
+4. When prompted enter link to wikimedia: http://sorenj02.nyc.corp.google.com/mediawiki/index.php/Main_Page and follow prompts
```

Comment:
Since this documentation is public, we might want to rephrase things a little to be something more like "enter the URL for your wikimedia site, for example http://sorenj02.nyc.corp.google.com/mediawiki/index.php/Main_Page".  That site is available internal to Google, but if anyone outside of Google is trying this (since it's open sourced) they will need their own wikimedia server.